Source/WebCore/css/process-css-pseudo-selectors.py

```diff
@@ -435,37 +435,37 @@ def prefix_value(prefix, value):
     def write_parsing_function_definitions_for_pseudo_class(self, writer):
         longest_keyword_length = len(max(self.mapping, key=len))
         writer.write_block("""
-        static inline const SelectorPseudoClassOrCompatibilityPseudoElementEntry* findPseudoClassAndCompatibilityElementName(const LChar* characters, unsigned length)
+        static inline const SelectorPseudoClassOrCompatibilityPseudoElementEntry* findPseudoClassAndCompatibilityElementName(std::span<const LChar> characters)
         {
-            return SelectorPseudoClassAndCompatibilityElementMapHash::in_word_set(reinterpret_cast<const char*>(characters), length);
+            return SelectorPseudoClassAndCompatibilityElementMapHash::in_word_set(reinterpret_cast<const char*>(characters.data()), characters.size());
```

Comment:
Surprised you didn’t change this to take a `span` too, given it’s probably not widely used, so wouldn’t balloon the size of the pull request.

Reply:
I had trouble finding the definition of in_word_set, seems to be a gperf function. I'll try and find it again to see if I can modify its declaration.

Reply:
I don't think I can control the parameters of `in_word_set`. gperf seems to allow tweaking the name of the lookup function but not its parameters. If you know how to do it, please let me know.

Reply:
I see, I didn’t realize it was generated by `gperf`. Let’s not waste more time tweaking this.